lib/template/code_example_templates/js_example_template.html.erb

```diff
@@ -0,0 +1,7 @@
+<script><%= rendered_example %></script>
+<div class="codeBlock jsExample">
+  <div class="highlight">
+    <pre><%= code_example %></pre>
+  </div>
+</div>
+
```

Comment:
I like this as well and am beginning to think it should be extended to all the default markdown methods as well in the future